docs-crawler/README.md

```diff
@@ -0,0 +1,22 @@
+# Threepio Documentation Crawler
+
+This documentation web crawler is used to generate a set of static JSON files which we can use to help perform certain automatic translations between PyTorch, Tensorflow, and Tensorflow.js.
+
+## Installation
```

Comment:
Feel free to pull down the branch and install yourself. I know poetry is a bit less common than pip/venv but it seems fairly easy to install & use.